podman/__init__.py

```diff
@@ -1,4 +1,8 @@
 """Podman service module."""
 from podman.api_connection import ApiConnection
 
-__ALL__ = ["ApiConnection"]
+import podman.system
```

Comment:
Is this right? `__ALL__` contains `system` whereas this imports it under `podman`

Reply:
not quite sure what the intention was with defining __ALL__ here in the first place, but first it should be `__all__` and not `__ALL__`, and second this only impacts `import * from podman` which is a dangerous practice anyways

otoh I was expecting for `import podman` to actually import stuff, hence this approach